internal/controllers/router/tests/templates/import-external-network.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Network
+metadata:
+  name: external-network
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack
+    secretName: openstack-clouds
+  managementPolicy: unmanaged
+  import:
+    filter:
+      name: "${E2E_EXTERNAL_NETWORK_NAME}"
```

Comment:
This variable is a new dependency for the kuttl tests and I'll need to export it in my PR https://github.com/k-orc/openstack-resource-controller/pull/286.